src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -120,6 +126,14 @@ export class MeetingManager implements AudioVideoObserver {
     if (config.postLogConfig) {
```

Comment:
Do we even want to set this if config.logger is set?

Reply:
If we want to handle this, we need extra condition here to check. Seems not very necessary to me cause it will make the code more complicated.
I'm not sure actually. In my opinion I think, If we provide this parameter, we should make sure it works. The purpose of a parameter is to pass in the external variable and set the value of internal variable. Whether to use it or not should be determined in the function, here is the `initializeMeetingSession()`.
Let me know your idea.

Reply:
i don't think it is more complicated. We do not use these parameters so it would be strange to set it and can cause bugs if other code read this properties and use it.

Reply:
Yeah, make sense, I will update it!